linux_cross.nix

```diff
@@ -0,0 +1,69 @@
+let
+  # Here we try to figure out which qemu to use based on the host platform.
+  # This guess can be overriden by passing qemuSuffix
+  qemuByHostPlatform = hostPlatform:
```

Comment:
There is a qemuArch param if you using a new enough nixpkgs. Just `hostPlatform.qemuArch` should work.

Reply:
This one currently misses `aarch64`, I think this would be great to use down the line.  For now I guess we have to handroll this (or just pull it from upstream nixpkgs) :-/